<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>685</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>877</width>
    <height>638</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>877</width>
    <height>700</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>10</y>
      <width>619</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>50</horstretch>
      <verstretch>50</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>224</width>
      <height>262</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="butClients">
       <property name="text">
        <string>Клиенты</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="butMonMaster">
       <property name="text">
        <string>Услуги маникюрного мастера</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="butKosmitolog">
       <property name="text">
        <string>Услуги косметолога</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="butUhodzaTelom">
       <property name="text">
        <string>Услуги по уходу за телом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="butStilist">
       <property name="text">
        <string>Услуги стилиста и визажиста</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="butMagazin">
       <property name="text">
        <string>Магазин</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>342</y>
      <width>249</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>57</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>К оплате:</string>
     </property>
    </widget>
    <widget class="QLabel" name="summa_uslug">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>8</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>41</y>
       <width>57</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>На счете:</string>
     </property>
    </widget>
    <widget class="QLabel" name="na_schetu">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>37</y>
       <width>89</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="but_schet">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>70</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Счет клиента</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="count_client">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>104</y>
       <width>37</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>108</y>
       <width>113</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Кол-во посещений:</string>
     </property>
    </widget>
    <widget class="QLabel" name="dolg_client">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>136</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="l_dolg">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Долг клиента:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_vnesti_dolg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>164</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Внести сумму</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>18</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Справочники</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="m_sprsotr"/>
    <addaction name="m_sprclients"/>
    <addaction name="m_spruslugi"/>
    <addaction name="m_spr"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Склад</string>
     </property>
     <addaction name="add_sklad"/>
     <addaction name="oststok_na_sklade"/>
     <addaction name="raspredel"/>
    </widget>
    <widget class="QMenu" name="menu_8">
     <property name="title">
      <string>Касса</string>
     </property>
     <addaction name="kassa_rashod"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="menu_8"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>отчеты по складу</string>
     </property>
     <addaction name="otc_ostatok"/>
     <addaction name="otc_ostatok_kab"/>
    </widget>
    <addaction name="menu_7"/>
    <addaction name="otc_kassa"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Сервис</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="settings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_sprsotr">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/res/icons/4.ico</normaloff>:/res/icons/4.ico</iconset>
   </property>
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </action>
  <action name="m_sprclients">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/res/icons/4.ico</normaloff>:/res/icons/4.ico</iconset>
   </property>
   <property name="text">
    <string>Клиенты</string>
   </property>
  </action>
  <action name="m_spruslugi">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/res/icons/4.ico</normaloff>:/res/icons/4.ico</iconset>
   </property>
   <property name="text">
    <string>Услуги</string>
   </property>
  </action>
  <action name="m_spr">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/res/icons/4.ico</normaloff>:/res/icons/4.ico</iconset>
   </property>
   <property name="text">
    <string>Материалы</string>
   </property>
  </action>
  <action name="add_sklad">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/res/icons/28.bmp</normaloff>:/res/icons/28.bmp</iconset>
   </property>
   <property name="text">
    <string>Поступление на склад</string>
   </property>
  </action>
  <action name="oststok_na_sklade">
   <property name="text">
    <string>Остатки на складе</string>
   </property>
  </action>
  <action name="raspredel">
   <property name="text">
    <string>Распределение материала</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Обработки</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="otc_ostatok">
   <property name="text">
    <string>Остаток материала на общем складе</string>
   </property>
  </action>
  <action name="otc_ostatok_kab">
   <property name="text">
    <string>Остаток материала в кабинетах мастеров</string>
   </property>
  </action>
  <action name="otc_kassa">
   <property name="text">
    <string>Кассовый отчет</string>
   </property>
  </action>
  <action name="kassa_rashod">
   <property name="text">
    <string>Расход </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
